corehq/apps/export/management/commands/delete_exports.py

```diff
@@ -33,6 +38,33 @@ def handle(self, domain, **options):
             print("No exports to delete here, exiting.")
             return
 
+        if options['days_inactive'] > 0:
+            import datetime
+            inactive_since = datetime.datetime.today() - datetime.timedelta(days=int(options['days_inactive']))
+            inactive_exports = []
+            for export in exports:
+                e = ExportInstance.get(export['id'])
+                if e.last_accessed and e.last_accessed <= inactive_since:
+                    inactive_exports.append(e)
+            if not inactive_exports:
+                print("No exports have been inactive for more than {days_inactive} days, exiting.".format(
+                    days_inactive=options['days_inactive'])
+                )
+                return
+            confirm = input(
+                "There are {total_exports} exports in {domain}. Are you sure you want to delete "
+                "{total_inactive_exports} that are older than {days_inactive} days [y/N]?".format(
+                    total_exports=len(exports),
+                    total_inactive_exports=len(inactive_exports),
+                    domain=domain,
+                    days_inactive=int(options['days_inactive'])
+                )
+            )
+            if confirm.lower() == 'y':
+                exports = inactive_exports
```

Comment:
I think you're mixing types here - `exports` is a result from couch with an `id` and a `key`, and `inactive_exports` is a wrapped `ExportInstance`.  I think line 68 would fail for inactive exports.

Reply:
You're absolutely right, thanks for catching that!